tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/GeneratorTests/sandwich/en-us/sandwichTest-Bread.en-us.lg

```diff
@@ -1,4 +1,6 @@
-
+[sandwichTest-BreadEntity.lg](sandwichTest-BreadEntity.lg)
+[sandwichTest-library.lg](sandwichTest-library.lg)
+[sandwichTest-library-Missing.lg](sandwichTest-library-Missing.lg)
```

Comment:
I've already changed the generator to do this and will be updating all of these tests soon.  Probably still need to do it anyway.